config/desktop-development.json

```diff
@@ -135,15 +134,12 @@
 		"upgrades/checkout": true,
 		"upgrades/credit-cards": true,
 		"upgrades/domain-search": true,
-		"upgrades/in-app-purchase": false,
```

Comment:
Added in https://github.com/Automattic/wp-calypso/commit/1cd15c830b, but I can't find where it was used. Maybe it was never used?

Reply:
Yeah, the usage never got merged: https://github.com/Automattic/wp-desktop/pull/17